.travis.yml

```diff
@@ -1,26 +1,42 @@
 language: generic
+dist: trusty
```

Comment:
Having the version more updated was something I thought about. In any case, there is no need, I guess.